services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> QuerySet[Movie]:
+    movies = Movie.objects.all()
+    if genres_ids:
+        movies = movies.filter(genres__in=genres_ids)
+    if actors_ids:
+        movies = movies.filter(actors__in=actors_ids)
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
Consider handling the `DoesNotExist` exception when using `get()` to retrieve a movie by ID. This will prevent the application from crashing if the movie ID does not exist.